src/hb-ot-hmtx-table.hh

```diff
@@ -79,6 +79,7 @@ struct hmtxvmtx
     hb_blob_t *src_blob = OT::Sanitizer<H>().sanitize (plan->source->reference_table (T::tableTag));
     if (unlikely(!src_blob))
     {
+      free(dest);
```

Comment:
Space before '(' please!

Reply:
free is gone courtesy of copy writeable fn